third-party/gasnet/GASNet-1.26.0/other/amudp/amudp_spmd.cpp

```diff
@@ -565,18 +565,25 @@ extern int AMUDP_SPMDStartup(int *argc, char ***argv,
       //
       // Create a new argv with space for -E options for the env vars.
       //
-      int new_argc = 4 + *argc;
+      int new_argc = 4 + *argc + 1;
       char** new_argv = (char**) AMUDP_malloc(new_argc * sizeof((*new_argv)));
 
       //
       // Instead of running the program directly, run an xterm which
-      // runs gdb on the program and its args.
+      // runs lldb on the program and its args.
       //
+#if defined __APPLE__
+      new_argv[0] = (char *) "/opt/X11/bin/xterm";
+      new_argv[1] = (char *) "-e";
+      new_argv[2] = (char *) "lldb";
```

Comment:
We need to be backward compatible, so it's a little tricky supporting lldb in addition to gdb because of the need for a different command line. I'm okay with adding CHPL_COMM_USE_LLDB, so I'll proceed along that route.